Source/OrchBase/build_defs.cpp

```diff
@@ -15,6 +15,8 @@ const string MAKEDIR = "mkdir";
 const string REMOVEDIR = "rm --force --recursive";
 #endif
 
+const unsigned int MAX_RTSBUFFSIZE = 4096;
```

Comment:
Not a fan of `define` preprocessing?

Reply:
I love `#define`'s, I was trying to be consistent with the existing contents of the file...